scripts/pull_dashboard_and_alert_rules.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Clone the repository with sparse checkout
+git clone --sparse https://opendev.org/openstack/sunbeam-charms.git
```

Comment:
```suggestion
git clone --sparse --single-branch --depth=1 https://opendev.org/openstack/sunbeam-charms.git
```
just a small optimization